engine/render/src/render/material.cpp

```diff
@@ -100,6 +52,65 @@ namespace dmRender
         return (dmGraphics::VertexAttribute::VectorType) -1;
     }
 
+    static void SetVertexAttributeDefaultSettings(dmGraphics::VertexAttribute* attribute, dmhash_t name_hash, dmGraphics::Type graphics_type, uint32_t element_count, bool instancing_supported)
```

Comment:
Consolidated a bunch of functions into a single one. Basically does the same thing as "attribute-key->default-attribute-info" in graphics.clj